### PR TITLE
Fix issue 380

### PR DESCRIPTION
**Pull request by mpoindexter**
Fix for https://github.com/aquasecurity/trivy-db/issues/380

### CODE REVIEW

File: pkg/vulnsrc/debian/debian_test.go [hunk 1]
```diff
@@ -104,6 +104,19 @@ func TestVulnSrc_Update(t *testing.T) {
 						FixedVersion: "19.4-2",
 					},
 				},
+				{
+					// Fix version not released yet
+					Key: []string{
```
Comment (DmitriyLewen) [inline pkg/vulnsrc/debian/debian_test.go:109]: Can you also add `vulnerability-detail` and `vulnerability-id` buckets for CVE-2023-5981?

**Review by DmitriyLewen — COMMENTED** [review]
@mpoindexter Hello!
Thanks for your work!

LGTM.
I left 1 small comment.

Regards, Dmitriy

File: pkg/vulnsrc/debian/debian_test.go (hunk 1 above)
Comment (mpoindexter) [reply pkg/vulnsrc/debian/debian_test.go:109]: @DmitriyLewen - done

**Review by DmitriyLewen — APPROVED** [review]
@mpoindexter Thanks for your work!

@knqyf263 I approved this PR. Take a look, when you have time.

**Comment by Zircon99** [comment]
Hi, do we have an ETA for when this will be ready? We have a pipeline dependent on this currently. 

Thanks

**Comment by knqyf263** [comment]
@mpoindexter Do you have time? If not, we'll take it over.

**Comment by mpoindexter** [comment]
Updated the PR to set the fix version to blank as requested

**Review by DmitriyLewen — APPROVED** [review]

**Review by knqyf263 — APPROVED** [review]

**Comment by knqyf263** [comment]
Thanks for your contribution!

**Pull request merged**